src/main/kotlin/uk/co/craigbass/pratura/boundary/ViewAllProducts.kt

```diff
@@ -0,0 +1,8 @@
+package uk.co.craigbass.pratura.boundary
+
+import com.madetech.clean.usecase.SynchronousUseCase
+import uk.co.craigbass.pratura.boundary.ViewAllProducts.PresentableProduct
+
+interface ViewAllProducts : SynchronousUseCase<Unit, List<PresentableProduct>> {
+  data class PresentableProduct(val name: String, val price: String, val sku: String)
+}
```

Comment:
Yeah I am unkeen on linting in compiled languages but even I switched on the IntelliJ auto end of file newline